src/miner.cpp

```diff
@@ -104,13 +104,6 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
     if (Params().MineBlocksOnDemand())
         pblock->nVersion = GetArg("-blockversion", pblock->nVersion);
 
-    // Create coinbase tx
-    CMutableTransaction txNew;
-    txNew.vin.resize(1);
-    txNew.vin[0].prevout.SetNull();
-    txNew.vout.resize(1);
-    txNew.vout[0].scriptPubKey = scriptPubKeyIn;
-
     // Add dummy coinbase tx as first transaction
```

Comment:
What do you mean? If you're referring to an input, coinbase transactions always have one input with a null prevout to indicate they are coinbases.